common.lic

```diff
@@ -392,8 +392,10 @@ module DRC
     g
   end
 
-  def play_song?(settings, song_list, worn = true)
+  def play_song?(settings, song_list, worn = true, skip_clean = false)
     instrument = worn ? settings.worn_instrument : settings.instrument
+    UserVars.song = song_list.first.first unless UserVars.song
+    fput('release ecry') if DRSpells.active_spells["Eillie's Cry"].to_i > 0
```

Comment:
💡 I suspect this could be simplified to `if DRSpells.active_spells["Eillie's Cry"]`. Does EC show up even if its inactive?

Reply:
I don't even get why it's in there at all, now that I look at it.  But I think it'll just show up as nil if it's not active, 0 if it's fading, and > 0 otherwise.  Just `if DRSpells.active_spells["Eillie's Cry"]` should work?

Reply:
That's my understanding, but I'd definitely test it with a Bard to be sure